lib/sidekiq/cron/job.rb

```diff
@@ -413,6 +412,12 @@ def to_hash
           last_enqueue_time: @last_enqueue_time.to_s,
           symbolize_args: symbolize_args? ? "1" : "0",
         }
+
+        if date_as_argument?
```

Comment:
could we add a test to cover this behaviour?

Reply:
I didn't know the best way to do this, but I added it! 503473b